tests/test_DAG_url_scraper.py

```diff
@@ -0,0 +1,81 @@
+from unittest import TestCase
+from dags.DAG_url_scraper import get_clean_urls
+
+
+class Test(TestCase):
+
+    def test_get_clean_tasks(self):
```

Comment:
imho a test should only test a specific behaviour / part of the logic. In that way you directly know where you have a problem - if the test fails. Better split this general test into multiple separate ones.